docs/src/modules/components/MaterialFreeTemplatesCollection.js

```diff
@@ -75,10 +77,11 @@ function layouts(t) {
 
 export default function Templates() {
   const t = useTranslate();
+  const theme = useTheme();
 
   return (
     <Grid container spacing={2} sx={{ py: 2 }}>
-      {layouts(t).map((layout) => (
+      {layouts(t, theme).map((layout) => (
```

Comment:
```suggestion
      {layouts(translatation, theme).map((layout) => (
```

I know this is out-of-scope from this PR but let's disambiguate this `t` variable here given we also have `theme`.